tests/Domain/Composer/ComposerFacadeTest.php

```diff
@@ -257,6 +257,34 @@ public function testIsValidConstraint($expected, $version): void {
     self::assertEquals($expected, $actual, 'Correctly determined validity of version constraint.');
   }
 
+  public function configList(): array {
```

Comment:
This is the convention in the rest of the tests:
```suggestion
  public function providerRemoveConfig(): array {
```
Also, the provider always goes _after_ the test method.